foxtrot-core/pom.xml

```diff
@@ -11,37 +11,88 @@
 
     <artifactId>foxtrot-core</artifactId>
 
+    <properties>
+        <hbase.version>1.2.1</hbase.version>
+        <zookeeper.version>3.4.8</zookeeper.version>
+    </properties>
+
     <dependencies>
-        <!--HBase and Hadoop-->
         <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-client</artifactId>
-            <version>2.0.0-mr1-cdh4.2.0</version>
+            <groupId>org.apache.hbase</groupId>
+            <artifactId>hbase-server</artifactId>
```

Comment:
@r0goyal The dependendency on hbase-server 1.1.2 always existed in the form of HBaseDS (HBaseWD) library dependency. HBaseWD uses org.apache.hadoop.hbase.mapreduce.TableInputFormat format which is present in hbase-server  library. Also this library was pulling in the entire library with no exclusions. Also the library was specified in compile scope which vastly reduces flexibility and tightly couples the hbase-server library version that is required. To solve this issue; I have migrated the HBaseDS library to hbase-server 1.2.1  and also made the scope as provided.